repomgr/repomgr.go

```diff
@@ -885,28 +885,30 @@ func processOp(ctx context.Context, bs blockstore.Blockstore, op *mst.DiffOp) (*
 			return nil, err
 		}
 
-		rec, err := lexutil.CborDecodeValue(blk.RawData())
-		if err != nil {
-			if errors.Is(err, lexutil.ErrUnrecognizedType) {
-				log.Warnf("failed processing repo diff: %s", err)
-				return nil, nil
-			}
-
-			return nil, err
-		}
-
 		kind := EvtKindCreateRecord
 		if op.Op == "mut" {
 			kind = EvtKindUpdateRecord
 		}
 
-		return &RepoOp{
+		outop := &RepoOp{
 			Kind:       kind,
 			Collection: parts[0],
 			Rkey:       parts[1],
 			RecCid:     &op.NewCid,
-			Record:     rec,
-		}, nil
+		}
+
+		rec, err := lexutil.CborDecodeValue(blk.RawData())
```

Comment:
I guess we could move the record decoding up outside of the repomgr, and just include the bytes in the RepoOp from here

Reply:
That sounds roughly correct. Let's have a quick discussion later today about what `repomgr` is responsible for.